jmix-search/search/src/test/java/io/jmix/search/exception/UnsupportedFileFormatExceptionTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.search.exception;
+
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class UnsupportedFileFormatExceptionTest {
```

Comment:
This test doesn't make much sense - it just checks message format.
It's better to have test that validates the FileProcessor actually throws this exception because of unsupported file.

Reply:
Done